conf/declaration.routes

```diff
@@ -1,8 +1,8 @@
 # Declaration type
 
-GET         /dispatch-location                   controllers.declaration.DispatchLocationController.displayPage(mode: models.Mode ?= models.Mode.Normal)
+GET         /dispatch-location                   controllers.declaration.DispatchLocationController.displayPage()
```

Comment:
simpler routes definition :+1:

Reply:
I think this is going to make it easy to forget that `mode` needs to be considered 👎